media_driver/agnostic/gen11/codec/media_srcs.cmake

```diff
@@ -19,5 +19,9 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 media_include_subdirectory(hal)
-media_include_subdirectory(kernel)
+if("${Full_Open_Source_Support}" STREQUAL "yes")
+  media_include_subdirectory(kernel_open)
```

Comment:
I don't like word "open" in this context. Can we, please, use "free" instead?

Reply:
Can you explain more details why prefer "free" to "open" ?